src/utils/date.js

```diff
@@ -95,7 +95,7 @@ export function formatDateWithOptions(val, options = {localeMatcher: 'best fit'}
 	return rtf.format(val);
 }
 
-export function formatDate(val, dateStyle = 'short', timeStyle = 'medium', locale = getCurrentLocale()) {
+export function formatDate(val,dateFormat,  dateStyle = 'short', timeStyle = 'medium', locale = getCurrentLocale()) {
```

Comment:
This function is used in other places and will break because of the argument order change

Reply:
By including `full` and `relative` to your `formateDateCustom` function you can remove it from here altogether, it's not used anyway.